cmd/daytona/config/const.go

```diff
@@ -78,6 +79,8 @@ func GetDocsLinkFromGitProvider(providerId string) string {
 		return "https://docs.gitness.com/administration/user-management#generate-user-token"
 	case "azure-devops":
 		return "https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows#create-a-pat"
+	case "aws-codecommit":
+		return "https://aws.amazon.com/codecommit/resources/"
```

Comment:
Please also add the required environment variables that the user must set on the server machine for this to work.

Reply:
Yup